cmake/CMakeLists.txt

```diff
@@ -520,9 +520,12 @@ include_directories(${LAMMPS_STYLE_HEADERS_DIR})
 ###########################################
 # Actually add executable and lib to build
 ############################################
+set(LAMMPS_SUFFIX "" CACHE STRING "Suffix to append to lmp binary and liblammps")
```

Comment:
Are we exploring every possibility or are we creating reasonable defaults for our users? That naming scheme has been here forever and is all over the manual. Something custom is outside of the scope of the official build system. You can always edit the CMakefile if you really need it.